Tasks/VsTest/versionfinder.ts

```diff
@@ -19,7 +19,62 @@ export async function locateVSTestConsole(testConfig): Promise<string> {
     return Promise.resolve(vstestExePath);
 }
 
-export function locateTestWindow(testConfig: models.TestConfigurations): Promise<string> {
+export class VSTestVersion {
+
+    constructor(public vstestExeLocation: string, public majorVersion: number, public minorversion: number, public patchNumber: number) {
+    }
+
+    isTestImpactSupported(): boolean {
```

Comment:
I don't a use of this in base since each of the extension has its own implementation.. Not sure how we can avoid it though...